docs/source/draft_export.rst

```diff
@@ -0,0 +1,245 @@
+.. _draft_export:
+
+Draft Export
+============
+
+TL;DR: Draft-export is a new version of export introduced in PyTorch 2.8 which
+will always produce a graph, even if there are potential soundness issues. It
+will also generate a report listing out all of the issues export runs into while
+tracing and provide additional debugging information. For custom operators that
+don't have fake kernels, it will also generate a profile which you can register
+to automatically generate a fake kernel.
+
+Have you ever tried to :func:`torch.export.export` a model, only to encounter a
+data-dependent issue? You fix it, but then run into a missing fake kernel
+problem. And after resolving that, you get hit with another data-dependent
+issue. You wonder to yourself, I wish there was a way I could just get a graph
+to play around with, and be able to view all the issues in one place so that I
+can fix them later…
+
+Say hello to your new friend, ``draft_export``!
```

Comment:
a bit colloquial but okay I guess =)
Could also say something like: ``draft_export`` to the rescue =)